src/Game.py

```diff
@@ -0,0 +1,29 @@
+from copy import deepcopy
+
+boards =	{
```

Comment:
some weird white space, maybe you install a linter

Reply:
I installed pylint. Perhaps it needs some configuration ?

Reply:
Seems not easy to use the pylint without installing python on my machine

Reply:
I think it was a lost tab ^^ removed it